src/main/java/core/DataValues.java

```diff
@@ -0,0 +1,57 @@
+package core;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * This class reads a properties file.
+ */
+public class DataValues {
+    public static Properties ENV_VALUE = new Properties();
+    public static Properties GET_VALUE = new Properties();
+
+    static {
+        try {
+            ENV_VALUE = read("environmentVariables.properties");
+            GET_VALUE = read("config.properties");
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    /**
+     * Gets the values inside a properties file.
+     *
+     * @return the read properties.
+     */
+    public static Properties read(final String path) throws IOException {
+        Properties properties = new Properties();
+        File file = new File(path);
+
+        if (!file.exists()) {
+            file.createNewFile();
+        }
+        try (InputStream reading = new FileInputStream(file)) {
+            properties.load(reading);
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
You should do something with the exception, not just printing it. You could throw it